tests/snuba/api/endpoints/test_organization_group_index_stats.py

```diff
@@ -67,6 +67,36 @@ def test_simple(self):
         assert "userCount" in response_data[0]
         assert "lifetime" in response_data[0]
         assert "filtered" in response_data[0]
+        assert "isUnhandled" not in response_data[0]
+
+    @with_feature("organizations:issue-stream-performance")
+    def test_unhandled(self):
+        self.store_event(
+            data={"timestamp": iso_format(before_now(seconds=500)), "fingerprint": ["group-1"]},
+            project_id=self.project.id,
+        )
+        group_a = self.store_event(
+            data={"timestamp": iso_format(before_now(seconds=1)), "fingerprint": ["group-a"]},
+            project_id=self.project.id,
+        ).group
+
+        self.login_as(user=self.user)
+        response = self.get_response(query="is:unresolved", groups=[group_a.id])
+
+        response_data = sorted(response.data, key=lambda x: x["firstSeen"], reverse=True)
+
+        assert response.status_code == 200
+        assert len(response_data) == 1
+        assert "title" not in response_data[0]
+        assert "hasSeen" not in response_data[0]
+        assert "stats" in response_data[0]
+        assert "firstSeen" in response_data[0]
+        assert "lastSeen" in response_data[0]
+        assert "count" in response_data[0]
+        assert "userCount" in response_data[0]
+        assert "lifetime" in response_data[0]
+        assert "filtered" in response_data[0]
+        assert "isUnhandled" in response_data[0]
```

Comment:
Should there be a test for the inverse case or is that just the typical path that other tests cover? Would `isUnhandled: False` be there?

Reply:
i added a line to the simple test to make sure unhandled isn't there if not flagged in. this new test checks to make sure it is there if it is flagged. it would be `isUnhandled : False` in this test, but given how the other tests are written, and what we care about, the actual value doesn't matter as much, just that it is showing up in the response.